src/UploadNetworkInterface.js

```diff
@@ -14,13 +14,34 @@ export class UploadNetworkInterface extends HTTPFetchNetworkInterface {
       : this.getJSONOptions(req)
     return fetch(this._uri, options);
   }
+  
+  isArrayOfFiles(list) {
+    if (Array.isArray(list)) {
+      let returnValue = false;
+      for (var key in list) {
+        if (list[key] instanceof File) {
+          returnValue = true;
+        } else {
+          return false;
+        }
+      }
+      return returnValue
+    }
+    return false;
+  }
 
   isUpload({ request }) {
     if (request.variables) {
       for (let key in request.variables) {
         if (request.variables[key] instanceof FileList) {
           return true
         }
+        if (request.variables[key] instanceof File) {
+            return true;
+          }
+        if (this.isArrayOfFiles(request.variables[key])) {
+          return true;
```

Comment:
The rest of this file doesn't use semicolons, probably following [Standard Style](http://standardjs.com)